pkg/shares/share_splitting.go

```diff
@@ -72,21 +72,10 @@ func SplitTxs(txs coretypes.Txs) (txShares []Share, pfbShares []Share, shareRang
 	return txShares, pfbShares, mergeMaps(txMap, pfbMap), nil
 }
 
-func SplitBlobs(cursor int, indexes []uint32, blobs []coretypes.Blob, useShareIndexes bool) ([]Share, error) {
-	if useShareIndexes && len(indexes) != len(blobs) {
-		return nil, ErrIncorrectNumberOfIndexes
-	}
+func SplitBlob(blob coretypes.Blob) ([]Share, error) {
```

Comment:
Would appreciate if we could start to add go docs for our public functions. Maybe we can go do that in a separate PR

Reply:
+1 we have https://github.com/celestiaorg/celestia-app/issues/1321 to track existing functions but I'll add godoc to this function b/c added in this PR